protocol/x/clob/ante/clob.go

```diff
@@ -130,6 +130,38 @@ func (cd ClobDecorator) AnteHandle(
 				log.Error, err,
 			)
 		}
+	case *types.MsgBatchCancel:
+		// MsgBatchCancel currently only processes short-term cancels right now.
+		// No need to process short term orders on `ReCheckTx`.
+		if ctx.IsReCheckTx() {
+			return next(ctx, tx, simulate)
+		}
+
+		ctx = log.AddPersistentTagsToLogger(
+			ctx,
+			log.Handler, log.MsgBatchCancel,
+		)
+
+		success, failures, err := cd.clobKeeper.BatchCancelShortTermOrder(
```

Comment:
i think theres a different flow between txs included in mempool and proposed operations which do matches. So proposed operations will be run in process proposal run tx i assume. But place short term, batch cancel and cancel are in the mempool i think so get treated differently and by putting in ante handler u execute before doing matching orders

Reply:
ok I see to effectively batch cancels are actually executed before transactions are run through the anteHandler since the orders exist in mem and not in state so this is ok to do in the case of short term orders?

just making sure I understand correctly